### DISCUSSION TITLE
Building error

### DISCUSSION THREAD

**Discussion by Zun4m1** [SoCuul/SCInsta#88] [Support]
Line 45 of build.sh pyzule: command not found   Fix??  It shows after creating IPA green text

**Comment by SoCuul** [comment]
Try running `pip3 install pyzule`

**Reply by Zun4m1** [reply]
make: No rule to make target 'clean' Stop.     Maybe something wrong with theos

**Reply by SoCuul** [reply]
Did you make sure you're in the right folder?
Try running `ls` and making sure you see a file there called `Makefile`
